### PR TITLE
complete PortPage functionality except few user input details

### PR DESCRIPTION
**Pull request by A00020061**

### CODE REVIEW

File: src/Components/portfolio/mainPage.js [hunk 1]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
   let list = localStorage.getItem("list");
-  console.log(list);
   if (list) {
     return JSON.parse(localStorage.getItem("list"));
   } else {
     return [];
   }
 };
+const GetCoinLocalStorage = () => {
+  let coins = localStorage.getItem("coin");
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:47]: const

File: src/Components/portfolio/mainPage.js [hunk 2]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
+  if (coins) {
+    return JSON.parse(localStorage.getItem("coin"));
+  } else {
+    return [];
+  }
+};
 
 function MainPage() {
   const [IsActive, setIsActive] = useState(false);
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:56]: use camelCase

File: src/Components/portfolio/mainPage.js [hunk 3]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
 const getLocalItems = () => {
   let list = localStorage.getItem("list");
-  console.log(list);
   if (list) {
     return JSON.parse(localStorage.getItem("list"));
   } else {
     return [];
   }
 };
+const GetCoinLocalStorage = () => {
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:46]: use camelCase

File: src/Components/portfolio/mainPage.js [hunk 4]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
     } catch (err) {
       setIsError(true);
     }
   };
 
-  useEffect(() => {
-    getCoinData();
-  }, []);
+  const handleRemove = (item) => {
+    console.log("clicked item here", item);
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:78]: no console

File: src/Components/portfolio/mainPage.js [hunk 5]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
-    setList([...Object.values(list), item]);
+  const handleList = (items) => {
+    if (items) {
+      let str = items.value.toString().toLowerCase();
+      coinList.map((item) => {
+        if (item.id === str) {
+          const newItem = { ...item, items };
+          setCoinDataHouse([...coinDataHouse, newItem]);
+        }
+      });
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:94]: are you sure this is the best way to do this?

File: src/Components/portfolio/mainPage.js [hunk 6]
```diff
@@ -2,50 +2,107 @@ import React, { useEffect, useState } from "react";
...
   };
 
   useEffect(() => {
     localStorage.setItem("list", JSON.stringify(list));
   }, [list]);
-
+  useEffect(() => {
+    localStorage.setItem("coin", JSON.stringify(coinDataHouse));
+  }, [coinDataHouse]);
+  console.log(coinDataHouse);
```
Comment (yoloOnTheBattlefield) [inline src/Components/portfolio/mainPage.js:105]: remove console

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/Components/portfolio/mainPage.js (hunk 5 above)
Comment (A00020061) [reply src/Components/portfolio/mainPage.js:94]: Hey bro, what is the best approach I can do here?

**Pull request merged**